### PR TITLE
[Vxlan] : adding show vnet/vxlan cmds

### PR DESCRIPTION
**Pull request by dzhangalibaba**
add some basic show cmds:
* show vnet brief
* show vnet name \<vnetname\>
* show vnet interfaces
* show vnet neighbors
* show vnet routes all
* show vnet routes tunnel
* show vxlan tunnel
* show vxlan name \<vxlanname\>

* some example as below:

```
admin@ASW-7005:~$ show vnet brief
vnet name    vxlan tunnel      vni  peer list
-----------  --------------  -----  ------------------
Vnet_2000    tunnel1          2000
Vnet_3000    tunnel1          3000  Vnet_2000,Vnet4000
```

```
admin@ASW-7005:~$ show vnet name Vnet_3000
vnet name    vxlan tunnel      vni  peer list
-----------  --------------  -----  ------------------
Vnet_3000    tunnel1          3000  Vnet_2000,Vnet4000

```
```
admin@ASW-7005:~$ show vnet interfaces
vnet name    interfaces
-----------  ------------
Vnet_2000    Ethernet1
Vnet_3000    Vlan2000

```
```
admin@ASW-7005:~$ show vnet neighbors
Vnet_2000    neighbor     mac_address    interfaces
-----------  -----------  -------------  ------------
             11.11.11.11                 Ethernet1
             11.11.11.12                 Ethernet1

Vnet_3000    neighbor     mac_address        interfaces
-----------  -----------  -----------------  ------------
             20.20.20.20  aa:bb:cc:dd:ee:ff  Vlan2000
```

```
admin@ASW-7005:~$ show vnet routes tunnel
vnet name    prefix          endpoint    mac address        vni
-----------  --------------  ----------  -----------------  -----
Vnet_2000    100.100.1.1/32  10.10.10.1
Vnet_3000    100.100.2.1/32  10.10.10.2  00:00:00:00:03:04


```
```
admin@ASW-7005:~$ show vnet routes all
vnet name    prefix          nexthop    interface
-----------  --------------  ---------  -----------
Vnet_2000    100.100.3.0/24             Ethernet52
Vnet_3000    100.100.4.0/24             Vlan2000

vnet name    prefix          endpoint    mac address        vni
-----------  --------------  ----------  -----------------  -----
Vnet_2000    100.100.1.1/32  10.10.10.1
Vnet_3000    100.100.2.1/32  10.10.10.2  00:00:00:00:03:04
```

```

admin@ASW-7005:~$ show vxlan tunnel
vxlan tunnel name    source ip    destination ip    tunnel map name    tunnel map mapping(vni -> vlan)
-------------------  -----------  ----------------  -----------------  ---------------------------------
tunnel1              10.10.10.10
tunnel2              10.10.10.10  20.10.10.10       tmap1              1234 -> 100
tunnel3              10.10.10.10  30.10.10.10       tmap2              1235 -> 200

```

```
admin@ASW-7005:~$ show vxlan name tunnel3
vxlan tunnel name    source ip    destination ip    tunnel map name    tunnel map mapping(vni -> vlan)
-------------------  -----------  ----------------  -----------------  ---------------------------------
tunnel3              10.10.10.10  30.10.10.10       tmap2              1235 -> 200


```
Signed-off-by: Dong Zhang d.zhang@alibaba-inc.com

### CODE REVIEW

**Comment by dzhangalibaba** [comment]
@prsunny

**Comment by prsunny** [comment]
Just a reminder to update command reference as well - https://github.com/Azure/sonic-utilities/blob/master/doc/Command-Reference.md

**Comment by dzhangalibaba** [comment]
retest this please

**Comment by dzhangalibaba** [comment]
retest this please

File: show/main.py [hunk 1]
```diff
@@ -2916,5 +2916,288 @@ def autorestart(container_name):
...
+
+    click.echo(tabulate(table, header))
+
+@vnet.command()
+def neighbors():
+    """Show vnet neighbors information"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+
+    header = ['<vnet_name>', 'neighbor', 'interfaces']
```
Comment (prsunny) [inline show/main.py:3019]: Can you also provide the` mac address `if its available in the attribute?

File: show/main.py [hunk 2]
```diff
@@ -2916,5 +2916,288 @@ def autorestart(container_name):
...
+#
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def vxlan():
+    """Show vxlan related information"""
+    pass
+
+@vxlan.command()
+@click.argument('vxlan_name', required=True)
+def name(vxlan_name):
+    """Show vxlan name <vxlan_name> information"""
```
Comment (prsunny) [inline show/main.py:3151]: Do you intend to have the `name `in command? Its different from the description.

File: show/main.py (hunk 1 above)
Comment (dzhangalibaba) [reply show/main.py]: added.
```
admin@ASW-7005:~$ show vnet neighbors 
Vnet_2000    neighbor     mac_address    interfaces
-----------  -----------  -------------  ------------
             11.11.11.11                 Ethernet1


Vnet_3000    neighbor     mac_address        interfaces
-----------  -----------  -----------------  ------------
             20.20.20.20  aa:bb:cc:dd:ee:ff  Vlan2000


```

File: show/main.py (hunk 2 above)
Comment (dzhangalibaba) [reply show/main.py:3152]: yes, the cmd is like : show vxlan name XXX, the description is not showing <>, correct it now.

**Comment by dzhangalibaba** [comment]
> Just a reminder to update command reference as well - https://github.com/Azure/sonic-utilities/blob/master/doc/Command-Reference.md

updated

**Pull request merged**